src/node_url.h

```diff
@@ -74,6 +74,16 @@ struct url_data {
   std::string href;
 };
 
+namespace table_data {
+extern const char* hex[256];
```

Comment:
```suggestion
extern const char* const hex[256];
```

(Aside -- using a single string of '%00\0%01\0%02\0...' and then accessing it as `hex[index * 4]` saves a bit more space and probably improves cache locality a bit.)

Reply:
I've changed it to a single string. But to be honest, I think the readability isn't increased much.

![image](https://user-images.githubusercontent.com/2842176/122008546-c6b9b500-cdeb-11eb-8729-5839b74b2f04.png)

Reply:
Right – I didn’t say it would increase readability :)